tribits/core/package_arch/TribitsPackageDependencies.cmake

```diff
@@ -0,0 +1,204 @@
+# @HEADER
+# ************************************************************************
+#
+#            TriBITS: Tribal Build, Integrate, and Test System
+#                    Copyright 2013 Sandia Corporation
+#
+# Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
+# the U.S. Government retains certain rights in this software.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met:
+#
+# 1. Redistributions of source code must retain the above copyright
+# notice, this list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright
+# notice, this list of conditions and the following disclaimer in the
+# documentation and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the Corporation nor the names of the
+# contributors may be used to endorse or promote products derived from
+# this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# ************************************************************************
+# @HEADER
+
+
+################################################################################
+#
+# TribitPackageDependencies.cmake
+#
+# Contains macros and functions related to defining internal package and
+# external package/TPL dependencies with a little bit of logic.
+#
+################################################################################
+
+
+include_guard()
+
+include(TribitsParseArgumentsHelpers)
+include(MessageWrapper)
+
+cmake_policy(SET CMP0011 NEW) # include() does policy push and pop
```

Comment:
I think the issue is that I am hitting this in the cmake -P script `TribitsPackageDependencies_UnitTests.cmake` because it does not include a call to `cmake_mimimum_required()`.  I will try putting in  `cmake_mimimum_required(VERSION 3.17)` in all of the TriBITS CMake -P scripts and take out all of these `cmake_policy(SET CMP0011 NEW)` calls.  (My guess is that this is why I added several other `cmake_policy()` calls like this as well over the years.

Reply:
Addressed in #498